docs/java_cli_for_managedgateway.rst

```diff
@@ -145,8 +145,8 @@ The gateway can invoke sendDeviceManageRequest() method to make the attached dev
 
 .. code:: java
 
-	managedGateway.sendGatewayManageRequest(typeId, deviceId, lifetime, true, true);
-	
+	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);
```

Comment:
You are right, its sendDeviceManageRequest(). so use the following code,

managedGateway.sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);

Can you point the exact wording where it says the lifetime, true, true part is not required? I'm not able to figure out.